Jenkinsfile

```diff
@@ -38,24 +39,38 @@ def buildManifest = {String manifest, String bitbake_image ->
     }
 
     stage("Setup bitbake and do fetchall ${bitbake_image}") {
-        // Extract the BSP part of the manifest file name.
-        String bsp = manifest.split("-")[1].tokenize(".")[0]
         // Setup bitbake environment and trigger a 'fetchall'
-        sh "vagrant ssh -c \"TEMPLATECONF=${yoctoDir}/sources/meta-pelux-bsp-${bsp}/conf /vagrant/vagrant-cookbook/yocto/fetch-sources-for-recipes.sh ${yoctoDir} ${bitbake_image}\""
+        confdir = "conf"
+        if (qtauto) {
+            confdir += "-qt"
+        }
+        templateconf="${yoctoDir}/sources/meta-pelux/meta-${bsp}-extras/${confdir} "
```

Comment:
This is pre-variants code, so should be updated to reflect the variant handling. There are commits for this in master/pyro.